README.Wii

```diff
@@ -0,0 +1,47 @@
+# SDL port to Nintendo GameCube/Wii
+
+This port uses the libogc library and is meant to be built with the devkitPPC
+toolchain, available from http://www.devkitpro.org
+
+
+## Installation instructions
+
+It is strongly recommended to only use the pre-built packages available from
+the devkitPro repository, which can be installed via the
+[pacman](https://devkitpro.org/wiki/devkitPro_pacman) tool. This is the only
+supported way of using libSDL on the Nintendo GameCube/Wii.
+
+
+## Build instructions
+
+If you need to hack on libSDL itself and you are aware that doing so can
```

Comment:
And again the language here is over the top. We discourage building things from source for the sake of building things from source. The point of what devkitPro do is to provide a unified environment available to all homebrewers which should hopefully encourage collaboration.

This whole section should just be replaced with instructions for building a package - i.e.

A libSDL package can be built by running the command `dkp-makepkg` in the appropriate subdirectory of the https://github.com/devkitPro/pacman-packages repo. This will build a .pkg.tar.zst package which can be installed with `(sudo) (dkp-)pacman -U <package file>. Once that's done you can edit the code in src/SDL-1.2.15 source /opt/devkitpro/wiivars.sh for wii or /opt/devkitpro/cubevars.sh for gamecube. Use make there to build and sudo -E make install to install it for testing purposes.